tests/tests/test-gas.ts

```diff
@@ -80,4 +80,49 @@ describeWithMoonbeam("Moonbeam RPC (Gas)", `simple-specs.json`, (context) => {
 
     expect(await contract.methods.multiply(3).estimateGas()).to.equal(21204);
   });
+
+  // Current gas per second is at 8M and our weight limit is 500ms.
+  // This computes to 4M gas per block.
+  // Current implementation is limiting block operation to ~0.65% of the block gas limit
```

Comment:
The comment in the Rust code says 75%.

Reply:
Fixed, I improved the code to include the right computation
I also changed the test to use the exact amount of gas instead of approximate percentages.